www/js/app.js

```diff
@@ -841,7 +848,8 @@ function initServiceWorkerMessaging () {
                     // Until we find a way to tell where it is coming from, we allow the request through on all controlled clients and try to load the content
                     console.warn('>>> Allowing passthrough of SW request to process Zimit video <<<');
                 }
-                handleMessageChannelMessage(event);
+                if (params.useLibzim) handleMessageChannelByWasm(event);
```

Comment:
I don't think this name is accurate. The other message channel also uses WASM, and there are also ASM versions of libzim (which we need to support also for browsers that can't handle WASM). So I'd suggest you rename this to `handleMessageChannelForLibzim`. Or another name if you prefer, but it would be good to reference libzim explicitly, to be clear about what is being used in that message channel.